src/data/episode.py

```diff
@@ -57,7 +62,7 @@ def __repr__(self):
     def get_episode(self):
         support = np.zeros((self.batch_size, self.support_size, self.max_len), dtype=self.dtype)
         query = np.zeros((self.batch_size, self.query_size, self.max_len), dtype=self.dtype)
-        artists = np.random.choice(self.dataset, size=self.batch_size, replace=False)
+        artists = self.random.choice(self.dataset, size=self.batch_size, replace=False)
```

Comment:
shouldn't this line use self.dataset.artists instead of self.dataset in the call?